bundle/edu.gemini.auxfile.workflow/src/main/java/edu/gemini/auxfile/client/AuxFileClient.java

```diff
@@ -21,15 +21,17 @@
  */
 public final class AuxFileClient implements AuxFileSystem {
 
-    private final int chunkSize;
+    private static final int FileChunkSize = 32 * 1024;
+
     private final AuxFileServer server;
 
-    public AuxFileClient(KeyChain kc, int chunkSize, String host, int port) {
-        this.chunkSize = chunkSize;
-        this.server    = TrpcClient$.MODULE$.apply(host, port).withKeyChain(kc).proxy(AuxFileServer.class);
+    public AuxFileClient(String host, int port) {
+        this.server    = TrpcClient$.MODULE$.apply(host, port).withoutKeys().proxy(AuxFileServer.class);
```

Comment:
Agreed. We need to introduce some sort of a service account that has staff privileges or some such thing.